tests/test_plugin.py

```diff
@@ -43,3 +43,48 @@ def test_plugin_create_tracker_store(
     manager.hook.create_tracker_store.assert_called_once_with(
         endpoint_config=endpoint_config, domain=domain, event_broker=event_broker
     )
+
+
+@pytest.mark.parametrize("endpoints_file", [None, "test_endpoints.yml"])
+def test_plugin_read_anonymization_rules(
+    endpoints_file: Optional[Text],
+    monkeypatch: MonkeyPatch,
+) -> None:
+    manager = plugin_manager()
+    monkeypatch.setattr(
+        manager.hook, "read_anonymization_rules", MagicMock(return_value=[])
+    )
+
+    manager.hook.read_anonymization_rules(endpoints_file=endpoints_file)
+    manager.hook.read_anonymization_rules.assert_called_once_with(
+        endpoints_file=endpoints_file
+    )
+
+
+@pytest.mark.parametrize(
+    "anonymization_rules, event_broker_config",
```

Comment:
This is trickier for `anonymization_rules` on OSS end, because we can't import the object type that would make up the `List` from `rasa-plus`. For `event_broker_config`, I'm testing with both `None` and `EndpointConfig` object type (we don't really need to pass args to this object).

Reply:
Yeah, I figured this out when I reviewed the other PR